modules/programs/tmux.nix

```diff
@@ -23,14 +51,125 @@ let
     };
   };
 
+  statusModule = types.submodule {
+    options = {
+      currentWindowFormat = mkOption {
+        default = "";
+        type = types.str;
+        description = ''
+          Current window status formatting.
+        '';
+      };
+
+      currentWindowStyle = mkOption {
+        default = "";
+        type = types.str;
+        description = ''
+          Current window style.
+        '';
+      };
+
+      justification = mkOption {
+        default = "centre";
+        type = types.str;
+        description = ''
+          How to justify statusbar info.
+        '';
+      };
+
+      keyMode = mkOption {
+        default = defaultKeyMode;
+        type = types.str;
+        description = ''
+          VI or Emacs style shortcuts.
+        '';
+      };
+
+      leftFormat = mkOption {
+        default = "";
+        type = types.str;
+        description = ''
+          Left status part formatting.
+        '';
+      };
+
+      leftLength = mkOption {
+        default = 30;
+        type = types.int;
+        description = ''
+          Length of left status part.
+        '';
+      };
+
+      messageStyle = mkOption {
+        default = "";
+        type = types.str;
+        description = ''
+          Message area style.
+        '';
+      };
+
+      rightFormat = mkOption {
+        default = "";
+        type = types.str;
+        description = ''
+          Left status part formatting.
+        '';
+      };
+
+      rightLength = mkOption {
+        default = 140;
+        type = types.int;
+        description = ''
+          Length of right status part.
+        '';
+      };
+
+      style = mkOption {
+        default = "";
+        type = types.str;
+        description = ''
+          Style.
```

Comment:
I think this description could be elaborated, I can't tell what it is for. But perhaps it is obvious for an experienced tmux user?